core/src/main/scala/cats/data/package.scala

```diff
@@ -142,4 +142,6 @@ package object data extends ScalaVersionSpecificPackage {
       RepresentableStoreT.pure[W, Function1[S, *], S, A](x)
 
   }
+
+  def absurd[A](n: Nothing): A = throw new Exception("Somehow instantiated a value of INothing, this is a bug.")
```

Comment:
you don't need to `throw`. since `Nothing <: A` you can just return `n`:

```scala
def absurd[A](n: Nothing): A = n
```

also, the name `INothing` has persisted.

Reply:
I'll likely simply delete this function - I believe I only use something like this in one place so it is probably no longer necessary here.